auth-manifest/types/src/lib.rs

```diff
@@ -0,0 +1,168 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+   lib.rs
+
+Abstract:
+
+    File contains data structures for the image authorization manifest bundle.
+
+--*/
+
+#![no_std]
+
+use core::ops::Range;
+
+use caliptra_image_types::*;
+use core::default::Default;
+use memoffset::span_of;
+use zerocopy::{AsBytes, FromBytes};
+use zeroize::Zeroize;
+
+pub const AUTH_MANIFEST_MARKER: u32 = 0x4154_4D4E;
+pub const AUTH_MANIFEST_IMAGE_METADATA_MAX_COUNT: usize = 16;
+pub const AUTH_MANIFEST_VENDOR_SIGNATURE_REQURIED_FLAG: u32 = 0x1;
+
+#[repr(C)]
+#[derive(AsBytes, FromBytes, Default, Debug, Clone, Copy, Zeroize)]
```

Comment:
Things other than private keys don't really need to be zeroized, but I suppose it doesn't hurt?

Reply:
This is due to a FIPS requirement for zeroizing local state.